packages/apps/job-launcher/server/src/modules/user/user.service.ts

```diff
@@ -100,4 +101,24 @@ export class UserService {
       currency: Currency.USD,
     };
   }
+
+  async createOrUpdateAPIKey(userId: number): Promise<string> {
+    const salt = crypto.randomBytes(16).toString('hex');
+    const apiKey = crypto.randomBytes(32).toString('hex');
+    const hashedAPIKey = crypto.pbkdf2Sync(apiKey, salt, 1000, 64, `sha512`).toString(`hex`);
```

Comment:
1000 and 64 can be added using configService.

Reply:
yes, added in config service!!